zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -532,6 +535,8 @@ final var record =
 
       camundaExporter.export(record);
 
+      controller.runScheduledTasks(Duration.ofSeconds(config.getBulk().getDelay()));
```

Comment:
❓ Why we schedule this so often?

Reply:
Refactored the test so there's only one `runScheduledTasks` to try and trigger the scheduled flush.